docs/containers-registries.conf.5.md

```diff
@@ -10,6 +10,7 @@ The CONTAINERS-REGISTRIES configuration file is a system-wide configuration
 file for container image registries. The file format is TOML.
 
 By default, the configuration file is located at `/etc/containers/registries.conf`.
+If user created file `$HOME/.config/containers/registries.conf` exists, will use user created file. If user's configuration file does not exist, will use `/etc/containers/registries.conf` as system-wide configuration path.
```

Comment:
How about
```
Container engines will use the `$HOME/.config/containers/registries.conf` if it exists, otherwise they will use `/etc/containers/registries.conf`